OOP/Assignment1/CarServiceSystem.java

```diff
@@ -0,0 +1,161 @@
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.Scanner;
+
+public class CarServiceSystem {
+	public static ArrayList<Mechanic> mech = new ArrayList<Mechanic>();
+	public static ArrayList<Car> car = new ArrayList<Car>();
+
+	public void showAllocation() {
+
+	Iterator<Car> iterator = ((ArrayList<Car>) car).iterator();
```

Comment:
ACTION: Better use "for" loop. You can search on net how to iterate over a list using for-loop.